### ISSUE TITLE
Rubbing/Dragging/Moving Alerts - Tested

### ISSUE THREAD

**Issue by terracoda** [phetsims/balloons-and-static-electricity#367]
Rubbing/Draggin/Moving Alerts Tested
March 12, 2018
March 12th, testing BASE from PhET Test
Voice Over and Safari (11.03) on Mac OS 10.13.3

**Rubbing against wall (Single balloon scenarios)**
For single balloon scenarios, the wall rubbing alert sounds fine in all charge views, except in the case of using JW to jump to the wall with _Show no charges_ checked.

Technically a Jumping Alert, I noticed that 
- [x] When I use JW to jump to Wall I get the wall's charge description. The walls charge information has to be suppressed when _Show no charges_ is selected.

This happens regardless of the number of active balloons.

**3A. Rubbing against wall (two-balloon scenarios)**
**Show all charges**
- Punctuation is good.
- Combined induced charge sounds great with two balloons (see related  issue #336 for Combined induced charge). 
- The next-to phrases sound great with two active balloons.
- [x] The un-grabbed balloon and related special cases for _Show all charges_ using the parameter "Each" are not yet implemented when balloons are next-to each other. 

**Show charge differences**
- [x] Punctuation (Show charge differences): Balloon's net charge description and wall's net charge description are missing periods.
- The charge information that is described is correct.
- Again, the next-to phrases sound great with two active balloons.
- [x] The un-grabbed balloon and related special cases for _Show charge differences_ are not yet implemented when balloons are next to each other.

**Updated Comment:** 
After discussing with @jessegreenberg, @terracoda decided to not to revisit two-balloon verbosity issues regarding describing the un-grabbed balloon with @emily-phet. (removed design question note).

Proceeding with descriptions of both balloon's being described when they are next to each other. See table in [Two-balloons Design Document section 3A](https://docs.google.com/document/d/1LrOWT4RZiS4emrumW479POK--zWveWiylZMd7unNyi0/edit#heading=h.y6nk7e1ep1um)

**Comment by terracoda** [comment]
@terracoda to discuss the design and implementation of describing the un-grabbed balloon with @jessegreenberg and @emily-phet before fully assigning to @jessegreenberg for implemention.

**Comment by terracoda** [comment]
@jessegreenberg and @terracoda discussed implementation overhead for describing the un-grabbed balloon and @jessegreenberg feels these are doable.

We will proceed with the un-grabbed balloon descriptions, as laid out in the design document.

**Comment by jessegreenberg** [comment]
> The un-grabbed balloon and related special cases for Show charge differences are not yet implemented when balloons are next to each other

There are issues with this, Ill  work on this next.

**Comment by jessegreenberg** [comment]
Everything in this issue has been addressed. Thanks @terracoda! Closing.
